### DISCUSSION TITLE
Dependenabot, update from private packages registry fails

### DISCUSSION THREAD

**Discussion by Aurelien30000** [community/community#24931] [Actions]
<p>Hey community,</p>
<p>I thought dependabot was able to access and update private dependencies (hosted on github packages registry) located under the same organization with required permissions. However, it fails everytime, I read some hypotesises about the fact it was reverted or cancelled or still not implemented, despite an announcement (<a href="https://github.blog/changelog/2020-12-02-dependabot-version-updates-from-private-github-repositories/" class="inline-onebox" rel="noopener nofollow ugc">Dependabot: version updates from private GitHub repositories - GitHub Changelog</a>)</p>
<p>I hope someone will help me sheding light on this :smiley:</p>
<p>Thanks!</p>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/aurelien30000">@Aurelien30000</a>,</p>
<aside class="quote no-group" data-username="Aurelien30000" data-post="1" data-topic="166221">
<div class="title">
<div class="quote-controls"></div>
 Aurelien30000:</div>
<blockquote>
<p>I thought dependabot was able to access and update private dependencies (hosted on github packages registry) located under the same organization with required permissions.</p>
</blockquote>
</aside>
<p>I’m afraid there isn’t currently any integration between Dependabot and GitHub Packages. At the moment the dependency graph only scans packages in public repositories. Hopefully support will be added for GitHub Packages, but I don’t have an ETA.</p>
